tests/unit/jobserver/views/test_workspaces.py

```diff
@@ -463,6 +463,26 @@ def get_repo_is_private(self, owner, name):
     assert response.context_data["show_publish_repo_warning"]
 
 
+def test_workspacedetail_authorized_toggle_notifications(rf):
```

Comment:
Might also be good to check that someone who's not a ProjectDeveloper has "user_can_toggle_notifications" set to False?